test/evergreen.yml

```diff
@@ -2136,6 +2136,40 @@ tasks:
           working_dir: "wiredtiger"
           command: bash test/evergreen/compatibility_test_for_wiredtiger_releases.sh
 
+  - name: format-stress-sanitizer-test-ppc
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          # CC is set to the system default "clang" binary here as a workaround. Change it back to mongodbtoolchain "clang" binary after BUILD-10248 fix.
+          configure_env_vars: CCAS=/opt/mongodbtoolchain/v3/bin/gcc CC=/usr/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
+          posix_configure_flags: --enable-diagnostic --with-builtins=lz4,snappy,zlib
+      - func: "format test script"
+        vars:
+          test_env_vars: ASAN_OPTIONS="abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=`ls /usr/bin/llvm-symbolizer* | tail -1`
```

Comment:
Can we set ASAN_OPTIONS the same way as task `format-stress-sanitizer-test`?

Reply:
Yeah, sure. I just followed the same test setting as per Jenkins.